src/functions/create/handler.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { formatJSONResponse } from '@libs/api-gateway';
+import { wrapWithConnection } from '@libs/lambda';
+
+import { ICreateProposalRequest } from '@models/create';
+import { createProposalOnDb } from '@services/proposals';
+import { LambdaError } from '@libs/errors';
+import createProposalSchema from './schema';
+
+const create: ValidatedEventAPIGatewayProxyEvent<typeof createProposalSchema> = async (event) => {
+  const {
+    partialTx,
+    authPassword,
+  } = event.body as ICreateProposalRequest;
+    // XXX: This is just to implement the error handling, but the schema should do this validation
```

Comment:
I didn't understand this comment, is the schema doing this validation or not? If positive, do we need this if statement?

Reply:
I was under the impression that the `json-schema-to-ts` dependency that is on the api-gateway middleware could later receive the responsibility of making the full validation of the inputs, something like `Joi` does.

But since then I found out this is out of their scope and the validation must be done on code as it is right now. In the future, maybe even add `Joi` as an additional validator to replace this handler logic.

Removed the comment.